cscs-checks/microbenchmarks/osu/osu_tests.py

```diff
@@ -32,14 +32,18 @@ def __init__(self, variant):
             'daint:gpu': {
                 'latency': (20.73, None, 2.0, 'us')
             },
+            '*': {
+                'latency': (0, None, None, 'us')
+            },
         }
         self.num_tasks_per_node = 1
         self.num_gpus_per_node  = 1
         if self.current_system.name == 'dom':
             self.num_tasks = 6
-
-        if self.current_system.name == 'daint':
+        elif self.current_system.name == 'daint':
```

Comment:
not an issue but why 6 and not 12 on dom:gpu ? and 16 on daint:gpu ? A comment would help

Reply:
@jgphpc I think that it is just because it's easier to get an allocation and also all the application tests on Dom are with 6 nodes.